docs/developer-center/new-guides/quickstart.md

```diff
@@ -1,185 +1,966 @@
+For our analysis, let's create and plot isochrones of 15 minutes by walk. Let's see how to do it:
 
-First, we will use the CARTO [data observatory]() to discover the data we want, data about households with no cars. You can learn more about discovering data in the [data discovery guide]().
 
-```py
-from cartoframes.data.observatory.catalog import Catalog
-from cartoframes.data import enrichment
+```python
+from cartoframes.data.services import Isolines
 
-countries = Catalog().countries()
-usa_datasets = countries.get("usa").datasets()
-acs_demo_tracts = usa_datasets.iloc[84]
-acs_no_car = acs_demo_tracts.variables().iloc[[88]]
+isochrones_df, _ = Isolines().isochrones(starbucks_df, [15*60], mode='walk')
+Map([
+    Layer(isochrones_df),
+    Layer(starbucks_df)]
+)
 ```
 
-Now that we have found the data we were looking for, let's filter out our area of intereset, Arlington. To do that, first we need to load Arlington's boundaries (they can be found [here](https://gisdata-arlgis.opendata.arcgis.com/datasets/census-tract-2010-polygons?geometry=-77.761%2C38.787%2C-76.772%2C38.974)):
+<div class="example-map">
+    <iframe
+        id="quickstart_guide_starbucks_brooklyn_iso"
+        src="https://cartoframes.carto.com/kuviz/17291073-7c93-431e-9dad-07e5da307a6f"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+</div>
 
-```py
-arlington_tracts = gpd.read_file('./data/tracts/1/Census_Tract_2010_Polygons.shp').to_crs({'init': 'epsg:4326'})
-```
+There they are! You can learn more about them and isodistances in the [location data services guide]().
 
-![Arlington tracts](../img/new-guides/quickstart/arlington_tracts.png)
 
-Let's continue enriching that area with the data we found before. To be able to do so, you have to log in to CARTO. You will need to create an API key and use the method `set_default_credentials` to create a session. If you haven't created an API key yet, check the [authentication guide]() to learn how to get it.
 
-Note: If you don't have an account yet, you can get a [free account]() if you are a student or [get a trial]() if you aren't.
+## Discover and enrich your data thanks to CARTO's Data Observatory
 
-```py
-from cartoframes.auth import set_default_credentials
+Once you know the areas of influence of your stores, let's try to find the population for each one. This will help us to calculate the average revenue per person.
+
+First, let's find the dataset we need. We will use the Catalog class that can be filter by country and category. In our case, we have to look for USA demographics datasets. Let's check which are the geographies (spatial resolution) availables.
```

Comment:
I think instead of "the dataset we need" it is more accurate to say "the demographic variable" or "the measure" just because it isn't actually a dataset but augmenting an existing dataset with this other information.

Reply:
replace: "Let's check which are the geographies (spatial resolution) availables."
with: "Let's check which geographies (spatial resolution) are available."